src/app/ui/overrides/adf-document-list.theme.scss

```diff
@@ -24,11 +42,19 @@
 
     .adf-datatable-cell-header,
     .adf-datatable-cell {
+      padding: 0 2px;
+      box-sizing: border-box;
+      position: static;
+
       &:focus {
-        outline: none !important;
+        //outline: none !important;
```

Comment:
if we don't need this style, we can remove entire `&: focus` block I think

Reply:
we do need that style, I have removed the comment. Thanks, Denys.